src/sbuf/SBuf.h

```diff
@@ -192,6 +193,9 @@ class SBuf
     /// Append a single character. The character may be NUL (\0).
     SBuf& append(const char c);
 
+    /// Append a single character, compatibly with std::string::push_back()
```

Comment:
Please drop the ", compatibly with std::string::push_back()" part. This method is _not_ compatible with std::string::push_back(), and we are not trying to be compatible with std::string::push_back(). We are trying to be compatible with std::back_inserter and similar STL code AFAICT, but that fact does not need to be documented explicitly IMO.

```suggestion
    /// Append a single character.
```

I am against sprinkling SBuf code with comments about NULs, but if you or others insist on preserving that part of the comment:

```suggestion
    /// Append a single character. The character may be NUL (\0).
```

I have removed the corresponding controversial text from the PR description.

Reply:
Corrected